modules/cloudbuild/main.tf

```diff
@@ -32,6 +32,17 @@ data "google_organization" "org" {
   organization = var.org_id
 }
 
+module "enable_cross_project_service_account_usage" {
+  source  = "terraform-google-modules/org-policy/google"
+  version = "~> 5.1"
+
+  project_id  = var.seed_project_id
```

Comment:
Can't we move this to the main module?

Reply:
@bharathkkb 
should it be always applied on the main module or should it be added it with a flag `enable_cross_project_service_account_usage` ?

Reply:
code moved to the main module